### PR TITLE
Jekyll plugins

### PR DESCRIPTION
**Pull request by pamo**
## Why?

A variety of things we might find useful
* [jekyll-seo-tag](https://github.com/jekyll/jekyll-seo-tag)
* [jekyll-sitemap](https://github.com/jekyll/jekyll-sitemap)
* [jemoji](https://github.com/jekyll/jemoji)

## What?
Basic configuration

## How can/should this be manually tested?

See details on each of the plugin's pages.

### CODE REVIEW

**Review by lizgene — CHANGES_REQUESTED** [review]
Where be your `Gemfile` & `Gemfile.lock`?

In case you ran `gem install gemname` - That will work locally, but you need to add the gem (preferably with a [version constraint](http://bundler.io/v1.5/gemfile.html)) to the `Gemfile` and then run a `bundle install`. That will update `Gemfile.lock` and you can commit them both.

File: _config.yml [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
-  Codify our practices and the reasoning behind them. Open source our study
-  guides and materials.
+  Open source SF Feminist Book Club study guides and materials and a registry of our practices and the reasoning behind them.
 baseurl: "/sf-feminist-book-club" # the subpath of your site, e.g. /blog
 url: "https://radicalbarracuda.github.io"
 permalink: /:categories/:title/
 # Build settings
+gems:
+  - jekyll-sitemap
+  - jemoji
```
Comment (lizgene) [inline _config.yml:11]: Clearly this is necessary for feminist book club study guides. 😂

**Comment by pamo** [comment]
Oh ooops. Didn't think those were necessary for gh-pages

**Comment by pamo** [comment]
@lizhubertz these are already present in `Gemfile.lock`

**Comment by lizgene** [comment]
@pamo Right you are 👍 🚢

**Review by lizgene — APPROVED** [review]

**Pull request merged**